pkg/cdi/cache_test_unix.go

```diff
@@ -0,0 +1,26 @@
+//go:build !windows
+// +build !windows
+
+/*
+   Copyright © 2021 The CDI Authors
```

Comment:
Nit: The date.

Reply:
Argh... does this non-functional change really warrant/require an update of the date ? I already regret including a date there at all. I'd much rather do another PR, dropping all dates from all files and the copyright template, similar to containerd.

Reply:
FWIW, the Kubernetes convention seems to be that the date is set once when the file gets added initially and then not updated anymore later on. It's just informative anyway, not legally binding.

Reply:
> FWIW, the Kubernetes convention seems to be that the date is set once when the file gets added initially and then not updated anymore later on. It's just informative anyway, not legally binding.

That sounds like a reasonable approach, too.